File: progra3-0583-equipo5-frontend/Frontend_C#/SoftPac/SoftPacPersistance/DAO/UsuarioPaisAccesoDAO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using SoftPac.Model;

namespace SoftPac.Persistance.DAO
{
    public interface UsuarioPaisAccesoDAO
    {
        int Insertar(UsuarioPaisAccesoDTO usuarioPaisAcceso);
        int Modificar(UsuarioPaisAccesoDTO usuarioPaisAcceso);
        int Eliminar(UsuarioPaisAccesoDTO usuarioPaisAcceso);
        UsuarioPaisAccesoDTO ObtenerPorUsuarioYPais(int usuarioId, int paisId);
    }
}

File: Backend_Java/SoftPacPersistance/src/main/java/pe/edu/pucp/softpac/dao/UsuarioPaisAccesoDAO.java
package pe.edu.pucp.softpac.dao;

import pe.edu.pucp.softpac.model.UsuarioPaisAccesoDTO;

public interface UsuarioPaisAccesoDAO {
    public Integer insertar(UsuarioPaisAccesoDTO usuario);

    public UsuarioPaisAccesoDTO obtenerPorUsuarioYPais(Integer usuarioId, Integer paisId);

    public Integer modificar(UsuarioPaisAccesoDTO usuario);

    public Integer eliminar(UsuarioPaisAccesoDTO usuario);
}
